pephub/routers/api/v1/project.py

```diff
@@ -1138,3 +1139,33 @@ def delete_full_history(
             status_code=400,
             detail="Could not delete history. Server error.",
         )
+
+
+@project.get(
```

Comment:
This _feels_ like it should be a `POST` request. I can't explain why but its doing an action (like a webhook) so it feels like it needs to be a `POST`

Reply:
updated